### ISSUE TITLE
Upload de arquivos integrado ao Plugin

### ISSUE THREAD

**Issue by jacsonp** [redelivre/delibera#32]
_From @jpmehl on April 15, 2013 20:6_

Hoje para carregar um arquivo no delibera é preciso instalar um plugin adicional (wp easy coment uploads). Precisamos ter esta funcionalidade como origem no plugin. 

_Copied from original issue: ethymos/delibera#10_

**Comment by jpmehl** [comment]
@lpirola acho que vc já resolveu isto, mas com plugin externo mesmo, certo? podemos fechá-la?

**Comment by lpirola** [comment]
podemos fechar sim!

**Comment by jpmehl** [comment]
Gente, reparei aqui testando a plataforma do CNPC que o upload de arquivo aparece também nos comentários gerais, conforme ver abaixo. Pode ver isto @lpirola ?
![upload em comentario de evento](https://cloud.githubusercontent.com/assets/4163139/19441044/f496b382-9462-11e6-8067-b98c7c01e9ce.png)

**Comment by lpirola** [comment]
@jpmehl Esse problema foi resolvido com o q vcs fizeram ou nada vê?

**Comment by Andreqlp** [comment]
@jpmehl isso foi resolvido?

**Comment by jacsonp** [comment]
Não, o upload assim como a menção continua dependente de um plugin externo
ao delibera

Em 18 de abr de 2017 6:51 PM, "André Quintino Lopes" <
notifications@github.com> escreveu:
